src/app/dataset-flow/dataset-flow-details/tabs/flow-details-summary-tab/flow-details-summary-tab.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="container">
+    <div class="row">
+        <div class="col pl-40">
+            <div class="row mb-4">
+                <div class="col-md-3"><span class="fw-bold">Flow type</span></div>
+                <div class="col-md-9" data-test-id="flow-type">{{ flowTypeDescription(flowDetails) }}</div>
+            </div>
+            <div class="row mb-4">
+                <div class="col-md-3"><span class="fw-bold">Status</span></div>
+                <div class="col-md-9" data-test-id="flow-status">{{ flowDetails.status | titlecase }}</div>
+            </div>
+            <div class="row mb-4">
+                <div class="col-md-3">
+                    <span class="fw-bold">Initiator</span>
+                </div>
+                <div class="col-md-9" data-test-id="flow-initiator">
+                    {{ flowDetails.initiator?.accountName ?? "System" }}
+                </div>
+            </div>
+            <div class="row mb-4">
+                <div class="col-md-3">
+                    <span class="fw-bold">Outcome</span>
+                </div>
+                <div class="col-md-9">
+                    <span
+                        [class]="outcomeClass(flowDetails.outcome)"
+                        *ngIf="flowDetails.outcome; else emptyBlock"
+                        data-test-id="flow-outcome"
+                        >{{ flowDetails.outcome | uppercase }}</span
+                    >
+                </div>
+            </div>
+            <div class="row mb-4">
+                <div class="col-md-3">
+                    <span class="fw-bold">Created time</span>
+                </div>
+                <div class="col-md-9">
+                    <span
+                        data-test-id="flow-created-time"
+                        *ngIf="flowDetails.timing.awaitingExecutorSince; else emptyBlock"
+                    >
+                        {{ flowDetails.timing.awaitingExecutorSince | date: "y-MM-dd, h:mm:ss a" }}</span
```

Comment:
We have a good tradition to declare constants for formats, so that we don't create many different formats

Reply:
Ok, took format from `AppValues`